src/components/VAudioTrack/VPlayPause.vue

```diff
@@ -2,9 +2,10 @@
   <VIconButton
     v-bind="$attrs"
     :tabindex="layout === 'box' ? -1 : 0"
-    class="play-pause flex-shrink-0 bg-dark-charcoal border-dark-charcoal text-white disabled:opacity-70"
+    class="play-pause flex-shrink-0 bg-dark-charcoal border-dark-charcoal text-white disabled:opacity-70 focus-visible:border-pink focus-visible:outline-none focus-visible:shadow-ring"
     :icon-props="{ iconPath: icon }"
     :aria-label="$t(label)"
+    variant="plain-dangerous"
```

Comment:
This isn't directly related to this PR but `plain-dangerous` is a confusing variant name, given that it can be used for a button that's not dangerous. It can be improved.

Reply:
Do you have a suggestion Dhruv? I suggested the `dangerous` name modeled more or less after similarly dangerous tools like React's `dangerouslySetInnerHTML`. I can see that it could be confused with a "dangerous" button (i.e., a button leading to a destructive action) but I can't think of any other terms that coveys the sense of "you probably shouldn't use this". `variant="none"` is an alternative one we could use but it doesn't convey the sense that the user needs to take on extra responsibility when using it to correctly implement accessible patterns.

Reply:
How about "plain--avoid"? The `--avoid` suffix can add that little bit of friction to discourage it's usage while not confusing with the "danger" convention we use for destructive actions.

Reply:
Works for me!

Reply:
Created #1181.